buildSrc/subprojects/binary-compatibility/src/main/groovy/gradlebuild.binary-compatibility.gradle

```
@@ -141,12 +147,29 @@ def checkBinaryCompatibility = tasks.register("checkBinaryCompatibility", Japicm
         excludedClasses = toPatterns(PublicApi.excludes + PublicKotlinDslApi.excludes)
 
         title = "Binary compatibility report for Gradle ${isSnapshot.get() ? "${baseVersion.get()}-SNAPSHOT" : version} since ${compatibilityBaselineVersion}"
-        destinationDir = file("$buildDir/reports/binary-compatibility")
-        reportName = "report.html"
+        destinationDir = binaryCompatibilityReportDir
+        reportName = binaryCompatibilityReport
     }
 
     BinaryCompatibilityHelper.setupJApiCmpRichReportRules(delegate, acceptedViolations, apiSourceFolders, baseVersion.get())
 }
+
+gradle.taskGraph.afterTask {
+    if (it.state.failure != null && it instanceof JapicmpTask) {
+        File report = new File(binaryCompatibilityReportDir, binaryCompatibilityReport)
+        Document doc = Jsoup.parse(report.text)
+        doc.select(".jumbotron").get(0).append("""
+<p>See the <a href="https://docs.google.com/document/d/1KA5yI4HL18qOeXjXLTMMD_upkDbNUzTDGNfBGYdQlYw/edit#heading=h.tfvlgk4slzc9">documentation</a> for more details.</p>
```

```suggestion
<p>See the <a href="https://docs.google.com/document/d/1KA5yI4HL18qOeXjXLTMMD_upkDbNUzTDGNfBGYdQlYw/edit#heading=h.9yqcmqviz47z">documentation</a> for more details.</p>
```

I think that is the correct heading.